master/usage/configuration/node.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Configuring a node IP address and subnet
+---
+
+By default, Calico automatically detects each Node's IP address and subnet.  In most cases, 
+this auto-detection is enough and you will not need to change the value picked by Calico.  
+However, there are some scenarios where you may want to configure these values explicitly
+rather than use auto-detection, or you may need to change which method of autodetection
+you use.  For example:
+
+-  Your host has multiple external interfaces
+-  Your host may have multiple IP addresses assigned to each interface
+-  You want to change the subnet configuration of each Node to use Calico's
+   [cross-subnet IPIP]({{site.baseurl}}/{{page.version}}/usage/configuration/ip-in-ip) feature.
+-  You have changed the IP of your host.
+
+This guide explains the various methods for configuring a Node's IP and subnet.
+
+### Using `calicoctl node run`
+
+If you are starting the calico/node container using `calicoctl node run` command, there is
+a direct mapping between the command line switches and the environment variables that are
+passed through to the calico/node container.  These are listed below:
+
+| Environment | CLI |
+|-------------|-----------------------------|-------------------|--------|
```

Comment:
Nit: A couple extra columns in this row, though it doesn't seem to affect the rendering any.